src/main/java/com/coveros/selenified/services/HTTP.java

```diff
@@ -255,26 +244,11 @@ public Response delete(String service, Request request) {
      * @return Response: the response provided from the http call
      */
     private Response call(String call, String service, Request request, File file) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS3776)